lib/collection.js

```diff
@@ -2867,7 +2867,7 @@ var mapReduce = function(self, map, reduce, options, callback) {
   };
 
   // Exclusion list
-  var exclusionList = ['readPreference', 'session'];
+  var exclusionList = ['readPreference', 'session', 'bypassDocumentValidation'];
```

Comment:
why does this need to be added to the exclusion list?

Reply:
I added it to the exclusion list because the loop after updates all the keys in mapCommandHash to be the options passed in, so when false is passed in, it will update bypassDocumentValidation to be false instead of undefined. I put byypassDocumentValidation in the exclusion list to prevent that. Should I delete bypassDocumentValidation from options or mapCommandHash if it's false instead?

Reply:
ah, that makes sense 👍